checks/included-plugins.php

```diff
@@ -0,0 +1,40 @@
+<?php
+class IncludedPlugins implements themecheck {
+	protected $error = array();
+
+	function check( $php_files, $css_files, $other_files ) {
+		$ret = true;
+
+		$filenames = array();
+
+		foreach ( $php_files as $php_key => $phpfile ) {
+			array_push( $filenames, strtolower( basename( $php_key ) ) );
+		}
+		foreach ( $other_files as $php_key => $phpfile ) {
+			array_push( $filenames, strtolower( basename( $php_key ) ) );
+		}
+		foreach ( $css_files as $php_key => $phpfile ) {
+			array_push( $filenames, strtolower( basename( $php_key ) ) );
+		}
+
+		$blacklist = array(
+			'\.zip'	=> __( 'Zipped Plugin', 'theme-check' ),
+		);
+
+		checkcount();
+
+		foreach( $blacklist as $file => $reason ) {
+			if ( $filename = preg_grep( '/' . $file . '/', $filenames ) ) {
+				$error = implode( array_unique( $filename ), ' ' );
+				$this->error[] = sprintf( '<span class="tc-lead tc-required">'.__( 'REQUIRED','theme-check').'</span>: '.__( '<strong>Zip file found.</strong> Plugins are not allowed in themes. The zip file found was ', 'theme-check') . '<em>' . __( $error ) . '</em>.' ) ;
```

Comment:
We can't translate variables `__( $error )`. As you are already using `sprintf` you can do `__( 'The zip file found was %s', 'theme-check' )` and then define `$error` as the counterpart to `%s`